tools/clang/test/CodeGenDXIL/hlsl/objects/NodeObjects/node-object-export-link-1.hlsl

```diff
@@ -1,42 +1,48 @@
 // RUN: %dxc -T lib_6_x %S/node-object-export-1.hlsl -Fo %t.1
 // RUN: %dxc -T lib_6_x %s -Fo %t.2
-// RUN: %dxc -T lib_6_x -link "%t.1;%t.2" | FileCheck %s
+// RUN: %dxc -T lib_6_x -link "%t.1;%t.2" -Fc %t.ll
+// RUN: FileCheck %s --input-file %t.ll --check-prefix=FOO2
+// RUN: FileCheck %s --input-file %t.ll --check-prefix=BAR
+// RUN: FileCheck %s --input-file %t.ll --check-prefix=BAR2
+// RUN: FileCheck %s --input-file %t.ll --check-prefix=BAR3
+// RUN: FileCheck %s --input-file %t.ll --check-prefix=BAR4
+// RUN: FileCheck %s --input-file %t.ll --check-prefix=FOO
 
 // Confirm that linking a shader calling external functions that take node objects as parameters
 // correctly includes the external and internal functions
 
 // Confirm that external function "foo2" is correctly included here even though it is called only by external functions
-// CHECK: define void @"\01?foo2@@YA?AU?$DispatchNodeInputRecord@URECORD@@@@U1@@Z"(%"struct.DispatchNodeInputRecord<RECORD>"* noalias nocapture sret, %"struct.DispatchNodeInputRecord<RECORD>"* nocapture readonly) #0 {
-// CHECK:   %[[Ld:.+]] = load %"struct.DispatchNodeInputRecord<RECORD>", %"struct.DispatchNodeInputRecord<RECORD>"* %{{.+}}, align 4
-// CHECK:   store %"struct.DispatchNodeInputRecord<RECORD>" %[[Ld]], %"struct.DispatchNodeInputRecord<RECORD>"* %{{.+}}, align 4
+// FOO2: define void @"\01?foo2@@YA?AU?$DispatchNodeInputRecord@URECORD@@@@U1@@Z"(%"struct.DispatchNodeInputRecord<RECORD>"* noalias nocapture sret, %"struct.DispatchNodeInputRecord<RECORD>"* nocapture readonly) #{{[0-9]+}} {
+// FOO2-NEXT:   %[[Ld:.+]] = load %"struct.DispatchNodeInputRecord<RECORD>", %"struct.DispatchNodeInputRecord<RECORD>"* %{{.+}}, align 4
```

Comment:
I wonder if these `NEXT` checks will be too strict. Can't think of a better way do it right now though.